src/systems/scene_broadcaster/SceneBroadcaster.cc

```diff
@@ -199,9 +199,18 @@ class ignition::gazebo::systems::SceneBroadcasterPrivate
   public: std::chrono::time_point<std::chrono::system_clock>
       lastStatePubTime{std::chrono::system_clock::now()};
 
+  /// \brief Period to publish state while paused and running. The key for
+  /// the map is used to store the publish period when paused and not
+  /// paused. The not-paused (a.ka. running) period has a key=false and a
+  /// default update rate of 60Hz. The paused period has a key=true and a
+  /// default update rate of 30Hz.
+  public: std::map<bool, std::chrono::duration<int64_t, std::ratio<1, 1000>>>
+      statePublishPeriod{{false, std::chrono::milliseconds(1000/60)},
+                         {true,  std::chrono::milliseconds(1000/30)}};
+
   /// \brief Period to publish state, defaults to 60 Hz.
   public: std::chrono::duration<int64_t, std::ratio<1, 1000>>
-      statePublishPeriod{std::chrono::milliseconds(1000/60)};
+      statePublishPeriodPaused{std::chrono::milliseconds(1000/30)};
```

Comment:
I think this is no longer used and safe to remove

Reply:
I believe this was needed for the GUI to be aware of the play/paused state so that it can enable/disable the edit boxes.

Reply:
I confirmed this. Change `itsPubTime` in line 306 below to include ` !_info.paused &&`. Run simulation and toggle the play/pause state. The edit boxes on the air pressure sensor will remain in the disabled state.

Reply:
hmm I think this variable is replaced by the `statePublishPeriod` map? I see in line 306 it's using the `statePublishPeriod` variable (as opposed to `statePublishPeriodPaused`). I also tried removing this variable and ign-gazebo still builds.

Reply:
Argh, right. My bad. Removed.